requirements.txt

```diff
@@ -16,3 +16,6 @@ black
 pytest
 coverage
 pytest-cov
+folium
```

Comment:
Update: I'd like to just ditch the interactive graph. It'd take me a while to refresh my bokeh knowledge and I feel like since we already have a static graph the interactive one is not a huge value add anyway.